ST_mortality.c

```diff
@@ -320,11 +320,7 @@ void mort_EndOfYear(void) {
           if (!Species[sp]->use_me){
             continue;
           }
-
           if (ZRO(Species[sp]->extragrowth)) continue;
-
-          Species[sp]->extragrowth = LE(Species[sp]->extragrowth, Species[sp]->relsize) ? Species[sp]->extragrowth : Species[sp]->relsize;
-          Species_Update_Newsize(sp, -Species[sp]->extragrowth);
           Species[sp]->extragrowth = 0.0;
         }
       }
```

Comment:
@chaukap just taking another careful look at this pull request before merging.

Now that `extragrowth` doesn't need to be removed from the species relsize here in `mort_EndofYear` when this year is a fire year, I don't think we need this function at all. At it appears to do is reset extragrowth to 0 if it is already zero.

   ```
   /* If the current year is a fire year, then remove extra_growth here
      instead of in _kill_extra_growth called in ST_main.c. Otherwise, 
      biomass will be non-zero in a fire year with complete killing */
      if (Globals.currYear == RGroup[rg]->killyr) {
        if (!RGroup[rg]->use_extra_res){
          continue;
        }

        ForEachGroupSpp(sp, rg, j) {
          /* If the species is turned off, continue */
          if (!Species[sp]->use_me){
            continue;
          }
          if (ZRO(Species[sp]->extragrowth)) continue;
          Species[sp]->extragrowth = 0.0;
        }
      }
```

Reply:
Since `kill_extra_growth` is called after `stat_Collect` removing this section would output a biomass greater than 0 in years with fire. Here's the flow:
```
      		mort_EndOfYear();
		stat_Collect(year);

		if (BmassFlags.yearly)
			output_Bmass_Yearly(year);

		_kill_annuals();
		proportion_Recovery();
		_kill_extra_growth();
```
We therefore need to remove extra growth here. so stat_Collect doesn't add extra growth.

Reply:
@chaukap ok great! thanks for the clarification.